code/cgame/cg_players.c

```diff
@@ -713,8 +713,13 @@ static void CG_SetColorInfo( const char *color, clientInfo_t *info )
 	CG_ColorFromChar( color[4], info->color2 );
 }
 
+static const int CG_ToBase7(int number){
+    if(number == 0)
+        return number;
+    return (number % 7) + 10*CG_ToBase7(number / 7);
+}
```

Comment:
Put whitespaces like John Carmack or some id dev coding style does. Sample:
```c
static const int CG_ToBase7( int number ) {
    if ( number == 0 )
        return number;
    return ( number % 7 ) + 10 * CG_ToBase7( number / 7 );
}
```
Since in that case, the code is readable enough.

Reply:
done